functions/__tests__/functions.test.ts

```diff
@@ -90,11 +90,11 @@ describe('extension', () => {
     test('function runs with created data', async () => {
       const beforeSnapshot = firebaseMock.firestore.makeDocumentSnapshot(
         {},
-        'collection/doc'
+        `collection/{}`
```

Comment:
Is it normal that it is empty ?

Reply:
Yes, it was to simulate a `firebaseMock.makeChange` object so I create a before (empty) and after snapshot (with data in it) and assemble it to create this object. Like:
```
      const beforeSnapshot = firebaseMock.firestore.makeDocumentSnapshot(
        {},
        `collection/{}`
      )
      const afterSnapshot = firebaseMock.firestore.makeDocumentSnapshot(
        defaultDocument.document,
        `collection/${defaultDocument.id}`
      )

      const documentChange = firebaseMock.makeChange(
        beforeSnapshot,
        afterSnapshot
      )
```
It may seem counterintuitive indeed